pkg/host/internal/sriov/sriov_test.go

```diff
@@ -38,10 +40,12 @@ var _ = Describe("SRIOV", func() {
 		testCtrl = gomock.NewController(GinkgoT())
 		netlinkLibMock = netlinkMockPkg.NewMockNetlinkLib(testCtrl)
 		dputilsLibMock = dputilsMockPkg.NewMockDPUtilsLib(testCtrl)
+		sriovnetLibMock = sriovnetMockPkg.NewMockSriovnetLib(testCtrl)
```

Comment:
can we add a unit-test to validate the vf representor return?

Reply:
I added one simple test for Discover function (positive scenario only)